core/src/main/java/nu/yona/server/exceptions/UserOverwriteConfirmationException.java

```diff
@@ -13,24 +10,46 @@
 public class UserOverwriteConfirmationException extends YonaException
```

Comment:
Can't we make a base class for `UserOverwriteConfirmationException` and `MobileNumberConfirmationException `?